src/main/java/org/folio/circulation/domain/Account.java

```diff
@@ -3,16 +3,35 @@
 import static org.folio.circulation.support.JsonPropertyFetcher.getNestedStringProperty;
 import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
 
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Optional;
+
+import org.apache.commons.lang.math.NumberUtils;
```

Comment:
Please use `org.apache.commons.lang3.math.NumberUtils`. We don't directly depend on the old commons-lang in the POM file.
```suggestion
import org.apache.commons.lang3.math.NumberUtils;
```

Reply:
Done.